Gemfile

```diff
@@ -7,3 +7,7 @@ gemspec
   local_gemfile = File.join(File.dirname(__FILE__), file_name)
   self.instance_eval(Bundler.read_file(local_gemfile)) if File.exist?(local_gemfile)
 end
+
+Dir[File.join(__dir__, 'gemfile.d', '*.rb')].each do |bundle|
```

Comment:
There's no need for this here. You can just define it in the Gemfile itself. Gemfile.d is for Foreman plugins.

While we don't use it today, dependabot doesn't support `eval_gemfile` so I'd stick to a plain Gemfile.

Though the instance_eval above already breaks dependabot.